README.md

```diff
@@ -88,6 +91,13 @@ To delete an `Email` you need:
 inbox.messages.first.delete
 ```
 
+Access to a private inbox
+```ruby
+inbox = Mailinator::Inbox.get('inbox-abcd1234', private_domain: true)
+```
+
+The 'download' and 'delete' work the same with private domain.
```

Comment:
I think it's worth to also add a section where it's explained that it is possible to pass any custom option, not only `private_domain`.

Reply:
Ok